README.md

```diff
@@ -22,15 +22,61 @@ Enhancement proposal.
 
 **NOT FULLY IMPLEMENTED**
 
-The latest commit of the master branch solely brings in the subset of the hostpath driver needed to validate 
-the basic in-line ephemeral volume scenario articulated below. 
-
+The latest commit of the master branch solely introduces both the `Share` CRD and the `projectedresoure.storage.openshift.io`
```

Comment:
Aside from fallout from the review of this PR, as well as the possible evolution at some point of the API closer toward https://kubernetes.io/docs/concepts/storage/volumes/#projected the other major items currently in my mind for follow on work:

1) persist the csi driver volume map to disk so that it can be re-populated after the driver has been recycled
2) the OLM install stuff
3) in conjunction, e2e tests

Reply:
I've now pushed changes for "persist the csi driver volume map to disk so that it can be re-populated after the driver has been recycled"

Given what we came up in planning today, items 2) and 3) will be handled in subsequent stories / PRs @adambkaplan @coreydaley @otaviof 

Also per planning we can discuss the current approach toward permissions here, but most likely will defer subsequent changes to other stories/PRs

PTAL at this one when you get the chance